demisto_sdk/commands/common/legacy_git_tools.py

```diff
@@ -0,0 +1,325 @@
+import os
+import re
+from typing import Callable, List
+
+import click
+from demisto_sdk.commands.common.constants import (KNOWN_FILE_STATUSES,
+                                                   TESTS_AND_DOC_DIRECTORIES,
+                                                   FileType)
+from demisto_sdk.commands.common.errors import Errors
+from demisto_sdk.commands.common.hook_validations.base_validator import \
+    BaseValidator
+from demisto_sdk.commands.common.tools import (filter_packagify_changes,
+                                               find_type, get_pack_name,
+                                               has_remote_configured,
+                                               is_origin_content_repo,
+                                               run_command)
+from demisto_sdk.commands.validate.validate_manager import ValidateManager
+
+
+def git_path() -> str:
+    git_path = run_command('git rev-parse --show-toplevel')
+    return git_path.replace('\n', '')
+
+
+def get_current_working_branch() -> str:
+    branches = run_command('git branch')
+    branch_name_reg = re.search(r'\* (.*)', branches)
+    if branch_name_reg:
+        return branch_name_reg.group(1)
+
+    return ''
```

Comment:
these funcs are defined twice. :bydesign:?

Reply:
you mean withing the git_util and the legacy? yes, we are yet to completely rid of the legacy git tools so we should have the option to get the branch name outside of the git util.